README.md

```diff
@@ -41,52 +41,54 @@ python setup.py install
 ### Introduction
 Create a `VPN` object for your management interface connection.
 ```python
-import openvpn_api.VPN
-v = openvpn_api.VPN('localhost', 7505)
+import openvpn_api
+c = openvpn_api.Connection('localhost', 7505)
+v = openvpn_api.VPN(c)
 ```
 
 Then you can either manage connection and disconnection yourself
 ```python
-v.connect()
+c.connect()
 # Do some stuff, e.g.
 print(v.release)
-v.disconnect()
+c.disconnect()
 ```
-If the connection is successful, `v.connect()` will return `True`.
-However, if the connection fails `v.connect()` will raise an `openvpn_api.errors.ConnectError` exception with the reason for the connection failure.
+If the connection is successful, `c.connect()` will return `True`.
+However, if the connection fails `c.connect()` will raise an `openvpn_api.errors.ConnectError` exception with the reason for the connection failure.
 
 Or use the connection context manager
 ```python
-with v.connection():
+with c.connection():
```

Comment:
This makes it inconvenient, as i would have to hold `Connection` instance (that i have no use for) and create `VPN` instance over and over.

What about making `VPN` able to connect when it needs to with provided `Connection`?  
Use case that i am thinking of is having rare queries to ovpnd management, so keeping socked opened all the time could be overkill.

Reply:
Of couse the user would use the connection if they needed to, because the `send_commend` function is inside the connection itself and the user may need to send a raw command.
Also the connection _must_ kept alive to be able to receive all events from ovpn interface.

Reply:
The VPN object is never supposed to get created multiple times; the user must create one and use it whenever they need it. actually it's possible to add a helper method that creates both Connection and VPN, for easier usage.

Reply:
For events, sure. But not everyone want/need to receive events. So far this lib was fine with sporadic use like 'what is the ovpn status right now' or 'send sigterm' with no need to maintain connection that is not used in the meantime.

Reply:
Maybe even without events, If someone wanted to occasionally fetch data from the ovpn, still holding a connection would perform slightly better, because the connection is already open and the only action required is to write something on that socket, instead of opening it again. An open and alive socket connection is not much overhead really.
But I agree that anyone using the library should decide themselves what approach they'll be using; maybe some other process also wants to connect to the interface and keeping an alive connection is not an option. And there should be an easy way (helper method) to do that.

Reply:
I'll commit the changes soon